packages/camel-catalog/package.json

```diff
@@ -13,7 +13,8 @@
     ".": "./dist",
     "./index.json": "./dist/index.json",
     "./types": "./dist/types/index.ts",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./*.json": "./dist/*.json"
```

Comment:
I didn't know this could be done :smile:

Reply:
I googled really hard to find this :rofl: